pymatgen/apps/borg/hive.py

```diff
@@ -69,7 +69,7 @@ class VaspToComputedEntryDrone(AbstractDrone):
 
     There are some restrictions on the valid directory structures:
 
-    1. There can be only one vasp run in each directory.
+    1. There can be only one vasprun in each directory.
```

Comment:
I would need to revert this. This doesn't look like a type. I assume this means a VASP job